NEWS.md

```diff
@@ -5,6 +5,10 @@
 * `seq_linter()` additionally lints on `1:n()` (from dplyr) 
   and `1:.N` (from data.table)  (#1396, @IndrajeetPatil).
 
+## Bug fixes
+
+* `get_source_expression()` no longer omits trailing non-code lines from knitr files (#1400, @AshesITR).
```

Comment:
get_source_expressions?

Reply:
That's the affected public facing function.

Reply:
i mean, is it a typo? i know we have an internal `get_source_expression`